src/libs/antares/args/args_to_utf8.cpp

```diff
@@ -0,0 +1,51 @@
+
+#include <yuni/yuni.h>
+#include "antares/args/args_to_utf8.h"
+
+
+#ifdef YUNI_OS_WINDOWS
+#include <string.h>
+#ifndef WIN32_LEAN_AND_MEAN
+#define WIN32_LEAN_AND_MEAN 1
+#endif
+#include <windows.h>
+#include <shellapi.h>
+#include <stdlib.h>
+#include <stdio.h>
+#endif
+
+#ifdef YUNI_OS_WINDOWS
+char** AntaresGetUTF8Arguments(int argc, char**)
+{
+    wchar_t** wargv = CommandLineToArgvW(GetCommandLineW(), &argc);
+    char** argvUTF8 = (char**)malloc(argc * sizeof(char*));
+    for (int i = 0; i != argc; ++i)
+    {
+        const uint len = (uint)wcslen(wargv[i]);
+        const uint newLen = WideCharToMultiByte(CP_UTF8, 0, wargv[i], len, NULL, 0, NULL, NULL);
+        argvUTF8[i] = (char*)malloc((newLen + 1) * sizeof(char));
+        memset(argvUTF8[i], 0, (newLen + 1) * sizeof(char));
+        WideCharToMultiByte(CP_UTF8, 0, wargv[i], len, argvUTF8[i], newLen, NULL, NULL);
```

Comment:
I think you could simplify a bit
```cpp
char** AntaresGetUTF8Arguments(int argc, char**)
{
#ifdef YUNI_OS_WINDOWS
// windows implementation here
#else
// non-windows implementation here
#endif
}
```

Reply:
Done, even if a class now replaces the 2 functions